proto/substrait/algebra.proto

```diff
@@ -725,6 +726,35 @@ message NestedLoopJoinRel {
   substrait.extensions.AdvancedExtension advanced_extension = 10;
 }
 
+// A delimiter join performs duplicate elimination on one side and then
+// pushes the rows with the duplicates eliminated into an arbitrary
+// number of scans on the opposing side to enact the join.  The keys are
+// used to implement the join condition.
```

Comment:
Once again, I think I don't understand.  Why would there be more than one scan?  How do scans enact a join?  Is this for a case where a single input is sent to two different join relations?  Or do we somehow need multiple scans to satisfy a single join?

Reply:
From my observations there is only one delimiter scan to generate the delimiter for matching.  The unclear language here is what probably should be considered an internal implementation detail (how to break up the task into smaller workloads).  I've updated this text.